runtime/testy/Ansible.py

```diff
@@ -0,0 +1,98 @@
+import socket
+import threading
+import time
+import fake_dawn
+import random
+import sys
+from runtimeUtil import *
+data = [0] #for testing purposes
```

Comment:
This variable is not used anywhere. (To use it inside a function, you should have a `global data` line at the top of the function)